apps/game_backend/lib/game_backend/items.ex

```diff
@@ -249,9 +249,8 @@ defmodule GameBackend.Items do
   Gets Item's purchase cost by given currency and item_template.
   Returns {:ok, purchase_cost} if found one.
   Returns {:error, :not_found} if there are none.
-  Fails if there are more than one.
   """
-  def get_purchase_cost_by_currency(currency_id, item_template) do
+  def get_item_purchase_cost_by_currency(currency_id, item_template) do
```

Comment:
Sorry for the nitpicking but while we're at it, this should be `get_item_template_purchase_cost_by_currency`. I'd also put the item_template as the first param, makes more sense and allows for piping from `get_item_template` or `item.item_template`

Reply:
Alright. Done!

Reply:
Great comment!